Lecture2/Task1/StringProcessing.java

```diff
@@ -0,0 +1,40 @@
+import java.util.Scanner;
+
+public class StringProcessing {
+    public static void main(String[] args) {
+
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.print("Enter a sentence: ");
+        String line = scanner.nextLine();
+
+        char sentence[] = new char[line.length()];
+        for(int i = 0; i < line.length(); i++){
+            sentence[i] = line.charAt(i);
+        }
+
+        System.out.print("\n" + "Your sentence in reverse order: ");
+        for(int i = line.length() - 1; i >= 0; i--){
```

Comment:
also you can use
String reverse = new StringBuffer(string).reverse().toString();